README.md

```diff
@@ -113,6 +113,24 @@ The full fsproj is detail below:
 
 Plugins for Myriad are supplied by simply including the nuget package in your project, the nuget infrastructure supplies the necessary MSBuild props and targets so that the plugin is used my Myriad automatically.  Further details for building plugins will follow but following the source for the fields plugin can be used as reference until I write further documentation.
 
+### Using your own Plugins
```

Comment:
This doesn't really flow well with the above notes that says 

> Further details for building plugins will follow

Isn't this at least in part, the further details? If so I'd think the further details comment could go below these new details.
.

Reply:
Good thinking, I can reword this